superset-frontend/package.json

```diff
@@ -256,7 +256,7 @@
     "@types/jquery": "^3.5.8",
     "@types/js-levenshtein": "^1.1.0",
     "@types/json-bigint": "^1.0.1",
-    "@types/mousetrap": "^1.6.11",
+    "@types/mousetrap": "^1.6.12",
```

Comment:
It's a good practice to have a separate PR for bumping a package unless it's related to the fix. This practice facilitates reverting only the package bump if needed.
```suggestion
```